services/blockchain-connector/config.js

```diff
@@ -35,11 +35,14 @@ config.brokerTimeout = Number(process.env.SERVICE_BROKER_TIMEOUT) || 10; // in s
  * External endpoints
  */
 config.endpoints.liskWs = process.env.LISK_APP_WS || 'ws://127.0.0.1:7887';
+config.endpoints.liskHttp =
+	process.env.LISK_APP_HTTP || config.endpoints.liskWs.replace('ws', 'http');
 config.endpoints.geoip = process.env.GEOIP_JSON || 'https://geoip.lisk.com/json';
 
 /**
  * API Client related settings
  */
+config.useHttpApi = Boolean(String(process.env.USE_HTTP_API).toLowerCase() !== 'false'); // Enabled by default
 config.isUseLiskIPCClient = Boolean(
```

Comment:
For consistency.
```suggestion
config.useHttpApi = Boolean(String(process.env.USE_HTTP_API).toLowerCase() !== 'false'); // Enabled by default
config.useLiskIPCClient = Boolean(
```
or
```suggestion
config.isUseHttpApi = Boolean(String(process.env.USE_HTTP_API).toLowerCase() !== 'false'); // Enabled by default
config.isUseLiskIPCClient = Boolean(
```